test/e2e/tests/phishing-controller/phishing-detection.spec.js

```diff
@@ -297,4 +330,151 @@ describe('Phishing Detection', function () {
       },
     );
   });
+
+  describe('Phishing redirect protections', function () {
```

Comment:
This test is not structured like our other tests simple for "best practice" reasons: tests set up _should_ be done in `before`/`beforeAll`, and teardown should be done in `after`/`afterAll`.

By making use of `before`/`beforeAll` this test sets up the browser fixtures only once for all of the tests. It still uses out `withFixtures` function, so it gets a bit awkward because this helper function requires an async callback and it returns `Promise` (for different and valid reasons, but you already know that).

Anyway, I don't mind redoing it if you hate it this way. I don't even particularly love the code-gymnastics I had to do to make `withFixtures` work in this test flow. It does make it faster though.